rocky/katalogus/views/plugin_settings_delete.py

```diff
@@ -55,7 +54,7 @@ def get_context_data(self, **kwargs):
 
     def get_success_url(self):
         return reverse(
-            "plugin_detail",
+            "boefje_detail",
             kwargs={
                 "organization_code": self.organization.code,
                 "plugin_type": self.plugin.type,
```

Comment:
Regarding the issue from @Darwinkel, my guess is that you still need to remove this line

Reply:
Yes, that indeed was the case.